examples/airbnb.temple

```diff
@@ -37,7 +37,6 @@ Property: service {
   Photo: struct {
     // implicit 'id' field
     // implicit 'parentID' field
-    // implicit 'createdBy' field
```

Comment:
Might need to update the shopping example too

Reply:
Can’t see any incorrect comments in shopping